java/examples/build.gradle

```diff
@@ -1,6 +1,7 @@
 plugins {
     // Apply the application plugin to add support for building a CLI application in Java.
     id 'application'
+    id "com.google.osdetector" version "1.7.3"
```

Comment:
Why do we need this? If we don't - please delete

Reply:
to make the example simpler for the end-user.

The user don't have to think about the classifier will auto-resolve/detect on line 14

Reply:
Also make life simpler for the developer, because if we have a placeholder on line 14, everyone will need to go there and change that to be able to build the project

Reply:
Oh I missed that it is in use there!
Thanks for the details.